package.json

```diff
@@ -49,5 +49,18 @@
   "lint-staged": {
     "*.{astro,cjs,js,jsx,mjs}": "eslint --cache --fix",
     "*": "prettier --cache --ignore-unknown --write"
-  }
+  },
+  "description": "Necesitarás tener instalado [Visual Studio Code](https://code.visualstudio.com/download), [Node.js 16+](https://nodejs.org/en/download/) y la extensión de Astro, la puedes encontrar [aquí](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)",
```

Comment:
You can delete lines 52 from line 65! <3

Reply:
Done